.travis.yml

```diff
@@ -1,5 +1,15 @@
+sudo: false
 language: go
 
+go:
+- "1.10"
+
+# Only clone the most recent commit.
+git:
+  depth: 1
+
+install: false
```

Comment:
Why is this false? If I believe the gist, this will try to go get dependencies. We want to enforce that someone only has to `go get` this repo and be ready to go. There was an instance when I tried to pick up the project again and the master build was breaking because travis was presumably updating dependencies and pulling them in, but it was missing from the `vendor/`

Reply:
See: https://docs.travis-ci.com/user/languages/go/#Dependency-Management 

This project has a Makefile so nothing is done by Travis during the install phase by default. It's assuming a make target will get called.

This can be confirmed by looking at the [rawlog](https://api.travis-ci.org/v3/job/393875206/log.txt) for a build:

```
travis_fold:start:install
[0Ktravis_time:start:0446f420
[0K$ true

travis_time:end:0446f420:start=1529363835115001582,finish=1529363835121119892,duration=6118310
[0Ktravis_fold:end:install
```

Looks like there is in fact no need to include this line at all since it's a no-op. I'll remove it.